### PR TITLE
New options: stream over HTTPS & stream with login

### PR DESCRIPTION
**Pull request by knackebrot**
Good for people with webif behind a reverse proxy

### CODE REVIEW

File: src/enigma2/Settings.h [hunk 1]
```diff
@@ -58,6 +58,8 @@ namespace enigma2
     const std::string& GetPassword() const { return m_strPassword; }
     bool GetAutoConfigLiveStreamsEnabled() const { return m_bAutoConfig; }
     int GetStreamPortNum() const { return m_iPortStream; }
+    bool GetUseSecureConnectionStream() const { return m_bUseSecureHTTPStream; }
+    bool GetUseLoginStream() const { return m_bUseLoginStream; }
```
[the remark this answers is not in the document]
Comment (phunkyfish) [reply src/enigma2/Settings.h]: All the other "Use" functions use the same double verb. So at the very least what's in the PR is consistent ;)

**Comment by phunkyfish** [comment]
Looks good to me

**Comment by phunkyfish** [comment]
@ksooo would you usually request the review changes be fixed up into the first commit or are you OK with a separate review commit?

**Comment by ksooo** [comment]
fixed up, please

**Comment by knackebrot** [comment]
squashed.

**Comment by phunkyfish** [comment]
Well done, all looks good now ;)

**Comment by ksooo** [comment]
Shouldn't the minor addon version number increased instead of the micro version number? This is a new feature, not a bug fix only release.

**Comment by phunkyfish** [comment]
Fair point should be: 3.13.0

**Comment by knackebrot** [comment]
We are now on v3.13.0 :)

**Comment by knackebrot** [comment]
I see that it failed on every platform I didn't try (I only tried MSVC). Should be fixed now. Thank you,

**Comment by phunkyfish** [comment]
Ok, you are looking good. Again 😉

**Comment by phunkyfish** [comment]
@ksooo are you Ok for me to approve and merge?

**Comment by ksooo** [comment]
Sure, go ahead. 

Please be aware that if you want the new version to appear in the new Kodi binary addon repo you need to tag the master HEAD after merge => 3.13.0-Leia

**Comment by phunkyfish** [comment]
Cool, will do. Thanks

**Pull request merged**